services/ipqsservicesample/ui/index.php

```diff
@@ -0,0 +1,257 @@
+              <button id="button-execute-api-scoring-address" type="button" class="btn btn-primary">Calculate</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row text-center m-1">
+        <h2 class="m-1 p-1 bg-secondary text-white rounded">
+          Calculated Results
+        </h2>
+        <div class="row mb-1">
+          <div class="col">
+            <textarea id="textarea-api-scoring-result" class="col-12" rows="10"></textarea>
+          </div>
+        </div>
+    </div>
+    
+    <!-- Include JQuery JavaScript Asset -->
+    <script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous"></script>
+    
+    <!-- Include Bootstrap JavaScript Asset -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL" crossorigin="anonymous"></script>
+    
+    <!-- Custom Page JavaScript -->
+    <script>
+      // Once the DOM is Fully Loaded
+      $(document).ready(function () {
+        const buttonSaveApiKeyToken = document.querySelector('#button-save-api-key-token');
+        const buttonClearApiKeyToken = document.querySelector('#button-clear-api-key-token');
+        const buttonShowApiKeyToken = document.querySelector('#button-show-api-key-token');
+        
+        const buttonExecuteApiScoringEmail = document.querySelector('#button-execute-api-scoring-email');
+        const buttonExecuteApiScoringPhone = document.querySelector('#button-execute-api-scoring-phone');
+        const buttonExecuteApiScoringUrl = document.querySelector('#button-execute-api-scoring-url');
+        const buttonExecuteApiScoringIp = document.querySelector('#button-execute-api-scoring-ip');
+        const buttonExecuteApiScoringAddress = document.querySelector('#button-execute-api-scoring-address');
+        
+        const inputApiKeyToken = document.querySelector('#input-api-key-token');
+        
+        const inputEmail = document.querySelector('#input-api-scoring-email');
+        const inputPhone = document.querySelector('#input-api-scoring-phone');
+        const inputUrl = document.querySelector('#input-api-scoring-url');
+        const inputIp = document.querySelector('#input-api-scoring-ip');
+        const inputAddress = document.querySelector('#input-api-scoring-address');
+        
+        const textareaApiScoringResult = document.querySelector('#textarea-api-scoring-result');
+        
+        const apiEndpoints = {
+          KEY: 'http://localhost:8081/ipqs/api/key',
+          EMAIL: 'http://localhost:8081/ipqs/score/email/',
+          PHONE: 'http://localhost:8081/ipqs/score/phone/',
+          URL: 'http://localhost:8081/ipqs/score/url/',
+          IP: 'http://localhost:8081/ipqs/score/ip/',
+        };
+        
+        let apiKeyToken = undefined;
```

Comment:
Let's go ahead and remove client-side API Key storage/management, as we've moved that over to the Go/Gin-based service. We can check if an API Key is set on initial page load by pinging our service, and if so, we'll update the UI messaging accordingly. It might be kind of cool to someday introduce event-based server => client autobinding between our service + UI, but for now it can stay simple + manually user-initiated.